tests/ansible/all.yml

```diff
@@ -1,6 +1,6 @@
-- include: setup/all.yml
+- import_playbook: setup/all.yml
```

Comment:
Note to self: what are implications of changing `include` to `import_playbook`? E.g. were we implicitly relying on some semantic behaviour? What is the earliest Ansible release with `import _playbook` (for desired 0.2 & 0.3 reunification)?

Reply:
Sorry, this was a fast shot from me.
I have reverted it 478f286e..ffb58cd5
include is deprecated, but until ansible 2.16 there is some time left.
I fully agree that this should be investigated first.
Maybe this would be a case for a new PR to fixx all deprecation warnings but stay compatible to ansible 2.9

Reply:
gosh darn it ... too many construction sites at the moment ... sorry, the general include statement is just deprecated but at the top level of a play import_playbook must be used, so I needed to revert the revert ...
https://github.com/ansible-community/ansible-build-data/blob/main/5/CHANGELOG-v5.rst#L708
_playbook - Error if using include instead of import_playbook_

Reply:
as far as I understand the docs, `include` at the top level of a play in ansible < 2.3 should behave the same way as `import_playbook` after 2.4.